app/views/landing/index.html.haml

```diff
@@ -0,0 +1,15 @@
+.container
+  .row.mt-4
+    .col 
+      %h1{:class => "text-center"} David's Blog
+  .row
+    .col 
+      %h2{:class => "text-center"} Welcome to my super feature-rich and responsive blog!
+  .row 
+    .col 
+      %h3{:class => "text-center"}= "Right now it is: #{@time}"
```

Comment:
The `=` and quotation marks are unnecessary.  You can interpolate directly into the text in `haml` 
```suggestion
      %h3.text-center Right now it is: #{@time}
```